allure-generator/src/main/javascript/plugins/testcase-retry/RetryView.hbs

```diff
@@ -3,7 +3,8 @@
     {{#each retries}}
         <div class="retry-row" data-uid="{{uid}}">
             <span class="label label_status_{{status}}">{{status}}</span>
-            {{date time.start}} at {{time time.start}}
+            <span>{{date time.start}} at {{time time.start}}</span>
+            <div class="retry__message">{{statusDetails}}</div>
```

Comment:
if you will use `<p>`, then you can avoid extra styles, because `<p>` provides margins by default